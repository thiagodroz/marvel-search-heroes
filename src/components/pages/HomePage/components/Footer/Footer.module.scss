@import 'styles/constants';

.Component {
  display: flex;
  justify-content: center;
  width: 100%;
  background-color: $primary-color;
}

.Text {
  color: #fff;
}
