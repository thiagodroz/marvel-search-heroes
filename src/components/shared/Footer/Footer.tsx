import React from 'react';

import { Container } from 'components/shared/Container';

import Styles from './Footer.module.scss';

export const Footer: React.FC = () => (
  <footer className={Styles.Component}>
    <Container>
      <strong className={Styles.Text}>Â© 2020 Feito por Thiago Nogueira</strong>
    </Container>
  </footer>
);
