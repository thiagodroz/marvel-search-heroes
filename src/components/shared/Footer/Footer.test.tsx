import React from 'react';
import { render } from '@testing-library/react';

import { Footer } from './Footer';

describe(`HomePages's Footer spec`, () => {
  it('should match last snapshot', () => {
    const { container } = render(<Footer />);

    expect(container).toMatchSnapshot();
  });
});
