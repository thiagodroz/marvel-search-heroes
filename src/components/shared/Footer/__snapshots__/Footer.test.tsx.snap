// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePages's Footer spec should match last snapshot 1`] = `
<div>
  <footer
    class="Component"
  >
    <div
      class="Component"
    >
      <strong
        class="Text"
      >
        Â© 2020 Feito por Thiago Nogueira
      </strong>
    </div>
  </footer>
</div>
`;
